packages/@okta/vuepress-site/conductor.yml

```diff
@@ -1362,9 +1362,41 @@ redirects:
   - from: /guides/sign-in-with-microsoft/index.html
     to: /docs/guides/sign-in-with-microsoft/
   - from: /guides/sign-into-mobile-app/index.html
-    to: /docs/guides/sign-into-mobile-app-redirect/
+    to: /docs/guides/build-custom-ui-mobile/android/main/
   - from: /guides/sign-into-mobile-app
-    to: /docs/guides/sign-into-mobile-app-redirect/
+    to: /docs/guides/build-custom-ui-mobile/android/main/
+  - from: /docs/guides/sign-into-mobile-app/create-okta-application/index.html
+    to: /docs/guides/build-custom-ui-mobile/android/main/
+  - from: /docs/guides/sign-into-mobile-app/create-okta-application
+    to: /docs/guides/build-custom-ui-mobile/android/main/
+  - from: /docs/guides/sign-into-mobile-app/ios/create-okta-application/index.html
+    to: /docs/guides/build-custom-ui-mobile/ios/main/
+  - from: /docs/guides/sign-into-mobile-app/ios/create-okta-application
+    to: /docs/guides/build-custom-ui-mobile/ios/main/
+  - from: /docs/guides/sign-into-mobile-app/android/create-okta-application/index.html
+    to: /docs/guides/build-custom-ui-mobile/android/main/
+  - from: /docs/guides/sign-into-mobile-app/android/create-okta-application
+    to: /docs/guides/build-custom-ui-mobile/android/main/
+  - from: /docs/guides/sign-into-mobile-app/react-native/create-okta-application/index.html
+    to: /docs/guides/build-custom-ui-mobile/react-native/main/#create-an-okta-application
+  - from: /docs/guides/sign-into-mobile-app/react-native/create-okta-application
```

Comment:
dup of line 2876 (goes to same location)

Reply:
I've deleted these redirects as they're supposed to go to a guide that does not exist yet.